src/main/java/pingis/controllers/LiveChallengeController.java

```diff
@@ -86,26 +95,111 @@ public RedirectView createTaskPair(String testTaskName, String implementationTas
 
     if (currentChallenge.getType() == ChallengeType.ARCADE) {
       redirectAttributes.addAttribute("realm", currentChallenge.getRealm().toString());
-      return new RedirectView("/playArcade");
+      return playArcade(redirectAttributes, currentChallenge);
     }
     redirectAttributes.addAttribute("taskId", testTask.getId());
     redirectAttributes.addAttribute("testTaskInstanceId", 0L);
-    return new RedirectView("/playTurn/" + currentChallenge.getId());
+    return playLive(currentChallenge, redirectAttributes);
```

Comment:
Very minor comment, the attributes are reversed in the two methods `playArcade` and `playLive`.